src/reducers/index.js

```diff
@@ -0,0 +1,83 @@
+import {CHANGE_CLASSROOM} from '../actions';
+import {GET_CLASSROOM_LIST_SUCCEDED} from '../actions';
+
+import {CHANGE_COHORT} from '../actions';
+import {GET_COHORT_LIST_SUCCEDED} from '../actions';
+
+import {GET_ASSIGNMENTS_LIST_SUCCEDED} from '../actions';
+
+import {GET_STUDENTS_LIST_SUCCEDED} from '../actions';
+
+const stateList = {
+    cohort: 1, 
+    cohortList: [], 
+    classroom: 1, 
+    classroomList: [],
+    assignmentsList: [],
+    studentsList: []
+}
```

Comment:
What do the 1's do? 

Btw I think I for suuuuure messed up this pull request somehow. Idk why this line is yellow